docs/intelligentapps/overview.md

```diff
@@ -72,7 +71,42 @@ You can also install the AI Toolkit extension manually from the Visual Studio Co
 > Check the **What's New** page after installation to see detailed features for each version.
 * After successful installation, the AI Toolkit icon appears in the Activity Bar.
 
-   ![Screenshot showing the AI Toolkit icon in the VS Code Activity Bar after installation](./images/overview/install-4.png)
+
+
+## Explore Azure AI Toolkit
+
+The Azure AI Toolkit opens in its own view, with the Azure AI Toolkit Icon now displayed on the VS Code Navbar. The extension has several main sections: My Resources, Model Tools, Agent and Workflow Tools, MCP Workflow, and Help and Feedback.
+
+   ![Screenshot showing the AI Toolkit Extension with highlighted sections.":::
```

Comment:
Something wrong with the syntax of the image link? Also, indentation is not needed here.